src/Compilers/Core/Portable/EmbeddedText.cs

```diff
@@ -365,15 +365,21 @@ public override void Write(byte[] array, int offset, int count)
                 // int.MaxValue chars so compression or encoding would have to be abysmal for
                 // this to overflow. We'd probably be lucky to even get this far but if we do
                 // we should fail fast.
-                checked { BytesWritten += count; }
+                checked
+                {
+                    BytesWritten += count;
+                }
```

Comment:
@sharwell ?

Reply:
This seems weakly desirable - if there is no pushback then keep the implementation the same way, otherwise just exclude the case. My guess is it's sufficiently rare that it won't get pushback.

Reply:
sgtm.